packages/core/src/types.ts

```diff
@@ -647,6 +648,10 @@ export interface StateNodeConfig<
    * @default false
    */
   preserveActionOrder?: boolean;
+  /**
+   * A description of the state node, rendered in Markdown
```

Comment:
We probably should mention here that "rendering in Markdown" applies to the XState visual tooling. It would also be nice to specify what flavor of Markdown this is interpreted as (I assume that it's GFM but could be Commonmark or something else)

Reply:
I removed the markdown part. Will specify in documentation for tooling, not here